pkg/controllers/inflightchecks/failedinit.go

```diff
@@ -19,49 +19,69 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/samber/lo"
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/utils/clock"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
 	"github.com/aws/karpenter-core/pkg/cloudprovider"
-	"github.com/aws/karpenter-core/pkg/controllers/machine"
+	"github.com/aws/karpenter-core/pkg/controllers/node"
 )
 
 // initFailureTime is the time after which we start reporting a node as having failed to initialize. This is set
 // so that we should have few if any false positives.
-const initFailureTime = time.Hour
+const initFailureTime = time.Minute
```

Comment:
is it right that we're changing this to 1 minute?

Reply:
Good catch. This was edited in testing. Missed changing it back